### PR TITLE
New: Add a `schema` property for validation

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

`connector-local` didn't got the schema with the other updates. I've marked as `New:`  but maybe it should be `Breaking` as the others so we can start fresh (and catch if something is wrong in the logs?) @antross what do you think?

### CODE REVIEW

**Comment by molant** [comment]
@antross this is ready. It's the remaining issue to get merged before we can release a new version 😊

File: packages/hint-manifest-icons/tsconfig.json [hunk 1]
```diff
@@ -11,5 +11,9 @@
     "include": [
         "src/**/*.ts",
         "tests/**/*.ts"
+    ],
+    "references": [
+        { "path": "../hint" },
+        { "path": "../utils-tests-helpers" }
```
Comment (antross) [inline packages/hint-manifest-icons/tsconfig.json]: Guess we missed this in the PR for `hint-manifest-icons`? Probably should keep this as a separate commit unless it doesn't matter since we haven't released it yet.

Comment (molant) [reply packages/hint-manifest-icons/tsconfig.json]: I thought I had pushed those changes yesterday to master :/

**Comment by molant** [comment]
@antross since it's only one commit I'll change the title when squashing via the UI. I'll look into the missing `tsconfig.json` references and where they are in master first though.

**Comment by molant** [comment]
@antross the change seems to be in https://github.com/webhintio/hint/commit/e834ef6cb302b593cbd18dc971c6853394096ed6. I'll rebase just in case

**Comment by molant** [comment]
Rebase worked. I'll go ahead and merge this.
